tests/unit/test_lxd.py

```diff
@@ -536,216 +530,7 @@ def call_effect(*args, **kwargs):
-        # Container was created before, and isn't running
-        self.fake_lxd.name = 'myremote:snapcraft-project'
-        self.fake_lxd.status = 'Stopped'
-
-        self.make_containerbuild().execute()
-
-        if hasattr(self, 'cross') and self.cross:
-            mock_container_run.assert_has_calls([
-                call(['snap', 'install', 'core', '--channel', 'stable']),
-                call(['snap', 'refresh', 'core', '--channel', 'stable']),
-                call(['snap', 'install', 'snapcraft', '--channel', 'edge',
-                      '--classic']),
-                call(['snap', 'refresh', 'snapcraft', '--channel', 'edge',
-                      '--classic']),
-            ])
-        else:
-            mock_container_run.assert_has_calls([
-                call(['snap', 'ack', '/run/core_123.assert']),
-                call(['snap', 'install', '/run/core_123.snap']),
-                call(['snap', 'ack', '/run/snapcraft_345.assert']),
-                call(['snap', 'install', '/run/snapcraft_345.snap',
-                      '--classic']),
-            ])
-
-    def test_command_with_sudo(self):
-        self.make_containerbuild().execute()
-        project_folder = '{}/build_project'.format(self.home)
-        args = ''
-        if self.target_arch:
-            args += ' --target-arch {}'.format(self.target_arch)
-        sudo = []
-        if self.user != 'root':
-            sudo = ['sudo', '-H', '-E', '-u', self.user]
-        self.fake_lxd.check_call_mock.assert_has_calls([
-            call(['lxc', 'exec', self.fake_lxd.name, '--'] + sudo
-                 + ['mkdir', '-p', project_folder]),
-            call(['lxc', 'exec', self.fake_lxd.name, '--'] + sudo
-                 + ['sh', '-c', 'cd {}; snapcraft snap --output snap.snap{}'.
-                    format(project_folder, args)]),
-        ])
-
-    @patch('os.getuid')
-    @patch('snapcraft.internal.lxd.Containerbuild._container_run')
-    def test_user_setup(self, mock_container_run, mock_getuid):
-        mock_container_run.side_effect = lambda cmd, **kwargs: cmd
-        mock_getuid.return_value = 1234
-
-        self.make_containerbuild().execute()
-
-        if self.euid > 0:
-            mock_container_run.assert_has_calls([
-                call(['useradd', self.user, '--create-home']),
-                call(['usermod', self.user, '-o', '-u', '1234', '-G', 'sudo']),
-                call(['chown', 'me:me', self.home]),
-            ])
-            self.fake_lxd.check_output_mock.assert_has_calls([
-                call(['lxc', 'exec', self.fake_lxd.name, '--',
-                      'tee', '-a', '/etc/sudoers'],
-                     input='{} ALL=(ALL) NOPASSWD: ALL\n'.format(
-                         self.user).encode()),
-                call(['lxc', 'exec', self.fake_lxd.name, '--',
-                      'sudo', '-H', '-u', 'me', 'ls',
-                      '{}/build_project'.format(self.home)]),
-            ])
-
-    def test_init_failed(self):
```

Comment:
Why are you dropping this test?

Reply:
because the `scenarios` all used remote remotes